crates/bevy_input/src/axis.rs

```diff
@@ -70,6 +74,11 @@ where
     pub fn remove(&mut self, input_device: T) -> Option<f32> {
         self.axis_data.remove(&input_device)
     }
+
+    /// Returns an iterator over all axes.
+    pub fn all_axes(&self) -> impl Iterator<Item = &T> {
```

Comment:
Any reason we can't have an `iter` method that returns an iterator over `(&T, &f32)`?

Reply:
Added.